test-framework/src/runner.rs

```diff
@@ -245,5 +245,9 @@ pub fn sanitize_output(out: &String) -> String {
     let mut leo_dir = std::env::current_dir().unwrap();
     leo_dir.pop();
     let leo_dir = leo_dir.into_os_string().into_string().unwrap();
+
+    dbg!(&leo_dir);
+    dbg!(&out);
```

Comment:
Needs to be removed before merging

Reply:
Yeah, I'm still pushing and testing how GH CI works. For some reason for one specific test outs are not sanitized.